packages/asset/contracts/Catalyst.sol

```diff
@@ -161,6 +189,14 @@ contract Catalyst is
         uint256 tokenId,
         string memory metadataHash
     ) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        require(
+            tokenId > 0 && tokenId < tokenCount,
```

Comment:
<= tokenCount ?

Reply:
tokenCount is a variable which keeps a tab on how many catalyst have been minted or could be minted

Reply:
so the token id should be less than that as this function just changes metadata hash

Reply:
other checks are <= ? Can you make the intended behaviour clear in the tokenCount tests pls

Reply:
I fixed that @atkinsonholly thanks

Reply:
I didn't understand what do you mean by min and max tokenCount test

Reply:
So if we have 5 types of Cats the tokenCount should be [x] and these checks should fail
Or if we have 1 type of Cat what happens etc